python_data_wrangling/.config/configstore/update-notifier-npm.json

```diff
@@ -0,0 +1,4 @@
+{
+	"optOut": false,
```

Comment:
git rm  python_data_wrangling/.assets

Reply:
Wouldn't removing the .assets folder break the README images?